soh/soh/util.cpp

```diff
@@ -303,6 +303,41 @@ std::vector<std::string> questItemNames = {
         "Gold Skulltula Token",
 };
 
+std::vector<std::string> rcareaPrefixes = {
+    "KF",
+    "LW",
+    "SFM",
+    "HF",
+    "LH",
+    "GV",
+    "GF",
+    "Wasteland",
+    "Colossus",
+    "Market",
+    "HC",
+    "Kak",
+    "Graveyard",
+    "DMT",
+    "GC",
+    "DMC",
+    "ZR",
+    "ZD",
+    "ZF",
+    "LLR",
+    "Deku Tree",
+    "Dodongos Cavern",
+    "Jabu Jabus Belly",
+    "Forest Temple",
+    "Fire Temple",
+    "Water Temple",
+    "Spirit Temple",
+    "Shadow Temple",
+    "Bottom of the Well",
+    "Ice Cavern",
+    "Gerudo Training Grounds",
+    "Ganon's Castle",
+};
+
```

Comment:
If the size of this will never change at runtime, it should be a `static const std::array<std::string, RA_MAX>` or something along those lines. Making it a std::vector means there's extra memory available at the end of this that will never be filled.

Reply:
Should I also replace the other vectors in this file? I used a vector to retain consistency with the other ones.

Reply:
We probably should, but not in this PR. I'm also assuming there's no other specific reason these are vectors, which I could be mistaken. So if you run into issues converting to std::array let me know.